providers/openebs/installers/operator/master/litmusbook/test.yaml

```diff
@@ -289,11 +289,11 @@
                   retries: 120
 
                 - name: Confirm if sparse pool is running in all the nodes
-                  shell: kubectl get pods -n {{ namespace }} -o jsonpath='{.items[?(@.metadata.labels.name=="cstor-sparse-pool")].status.phase}'
+                  shell: kubectl get pods -n {{ namespace }} -l openebs.io/storage-pool-claim==cstor-sparse-pool -o custom-columns=:status.phase --no-headers  
                   args:
                     executable: /bin/bash
                   register: sparse_pool_count
-                  until: (sparse_pool_count.stdout)|int == (maxpool_count.stdout)|int
+                  until: "(sparse_pool_count.stdout_lines)|length == (maxpool_count.stdout)|int  and 'Running' in sparse_pool_count.stdout"

```

Comment:
we need to check if all the pools are in running state only. This case may pass if the count matches and only one pool is running state. Other pool pods may be in some other state

Reply:
its done